Scripts/Editor/Tests/anvil-unity-tests.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "anvil-unity-tests",
+    "rootNamespace": "Anvil.Unity.Tests",
+    "references": [],
```

Comment:
We probably want to reference the Anvil asmdefs. 

`anvil-unity-core-editor`
`anvil-unity-core-runtime`
`anvil-csharp-core`

Reply:
By default, assembly definitions automatically reference other assemblies as needed, so it doesn't require an explicit list (i.e. I don't have to tell it to use the nunit DLL either). I figured that was fine for a testing assembly that isn't referenced by anything else.

Reply:
Oh that's good to know, fair enough!